client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -485,6 +542,17 @@ where
 		};
 		debug!(target: "rpc-spec-v2", "[follow][id={:?}] Subscription accepted", sub_id);
 
+		// Get all block hashes from the node's memory.
+		let info = self.client.info();
+		let finalized = HashAndNumber { hash: info.finalized_hash, number: info.finalized_number };
+		let Ok(InitialBlocks { in_memory, pruned_forks}) = get_initial_blocks_with_forks(&self.backend, finalized) else {
+			// Note: This could be warp sync error.
```

Comment:
Can you elaborate on the warp sync error? What did you encounter here?

Reply:
It might be possible for the tree route to fail because of a warp sync, similar to:  https://github.com/paritytech/substrate/blob/5288cd94ed3489b9420e0eeed5674940bc7de3f6/client/db/src/lib.rs#L1870-L1876


I've printed the error as suggested by davxy, thanks!